jenkins/internal-ci/centos-7.9.2009/pr/component_test/pyutils-premerge.groovy

```diff
@@ -0,0 +1,254 @@
+        BRANCH = "${ghprbTargetBranch != null ? ghprbTargetBranch : COMPONENTS_BRANCH}"
+        THIRD_PARTY_VERSION = "${OS_VERSION}-2.0.0-k8"
+        VERSION = "2.0.0"
+        PASSPHARASE = credentials('rpm-sign-passphrase') 
+		
+	    // Artifacts root location
+
+		// 'WARNING' - rm -rf command used on this path please careful when updating this value
+	    DESTINATION_RELEASE_LOCATION = "/mnt/bigstorage/releases/cortx/github/pr-build/${BRANCH}/${COMPONENT_NAME}/${BUILD_NUMBER}"
+        PYTHON_DEPS = "/mnt/bigstorage/releases/cortx/third-party-deps/python-deps/python-packages-2.0.0-latest"
+        THIRD_PARTY_DEPS = "/mnt/bigstorage/releases/cortx/third-party-deps/centos/${THIRD_PARTY_VERSION}/"
+        COMPONENTS_RPM = "/mnt/bigstorage/releases/cortx/components/github/${BRANCH}/${OS_VERSION}/dev/"
+        CORTX_BUILD = "http://cortx-storage.colo.seagate.com/releases/cortx/github/pr-build/${BRANCH}/${COMPONENT_NAME}/${BUILD_NUMBER}"
+
+        // Artifacts location
+        CORTX_ISO_LOCATION = "${DESTINATION_RELEASE_LOCATION}/cortx_iso"
+        THIRD_PARTY_LOCATION = "${DESTINATION_RELEASE_LOCATION}/3rd_party"
+        PYTHON_LIB_LOCATION = "${DESTINATION_RELEASE_LOCATION}/python_deps"
+		
+        build_id = sh(script: "echo ${CORTX_BUILD} | rev | cut -d '/' -f2,3 | rev", returnStdout: true).trim()
+        STAGE_DEPLOY = "yes"
+    }
+
+    stages {
+
+        // Build py_utils fromm PR source code
+        stage('Build') {
+            steps {
+				script { build_stage = env.STAGE_NAME }
+                script { manager.addHtmlBadge("&emsp;<b>Target Branch : ${BRANCH}</b>&emsp;<br />") }
+
+                sh """
+                    set +x
+                    echo "--------------BUILD PARAMETERS -------------------"
+                    echo "PY_UTILS_URL              = ${PY_UTILS_URL}"
+                    echo "PY_UTILS_BRANCH           = ${PY_UTILS_BRANCH}"
+                    echo "PY_UTILS_PR_REFSPEC       = ${PY_UTILS_PR_REFSPEC}"
+                    echo "-----------------------------------------------------------"
+                """
+                 
+                dir("cortx_utils") {
+
+                    checkout([$class: 'GitSCM', branches: [[name: "${PY_UTILS_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "${PY_UTILS_URL}",  name: 'origin', refspec: "${PY_UTILS_PR_REFSPEC}"]]])
+
+                    sh label: 'Build', script: '''
+                        yum install python36-devel -y
+                        ./jenkins/build.sh -v $VERSION -b $BUILD_NUMBER
+                        ./statsd-utils/jenkins/build.sh -v $VERSION -b $BUILD_NUMBER
+                        cd $PWD/py-utils/test
+                        python3 setup.py bdist_rpm --requires cortx-py-utils --version=1.0.0 --post-install test-post-install --post-uninstall test-post-uninstall
```

Comment:
Please request the utils team to provide script similar to `build.sh`

Reply:
okay

Reply:
Waiting for the script which comes from component team for now this is good to merge